scanpipe/templates/scanpipe/dropdowns/project_download_dropdown.html

```diff
@@ -31,6 +31,9 @@
       <a href="{% url 'project_results' project.slug 'attribution' %}" class="dropdown-item">
         <strong>Attribution</strong>
       </a>
+      <a href="{% url 'project_results' project.slug 'ort' %}" class="dropdown-item">
```

Comment:
I propose we add a `Other` or `Tools formats` drop-down next `Attribution` and place `ORT (package-list)` in there so if say in the near future we want to add support for [package.spdx.yml](https://github.com/oss-review-toolkit/ort/blob/main/plugins/package-managers/spdx/src/funTest/assets/projects/synthetic/libs/zlib/package.spdx.yml) or [project.spdx.yml](https://github.com/oss-review-toolkit/ort/blob/main/plugins/package-managers/spdx/src/funTest/assets/projects/synthetic/inline-packages/project-xyz.spdx.yml) formats or any other tool proprietary format then we have menu item to add these without cluttering up the user experience.